_data/simple-icons.json

```
@@ -3404,6 +3404,12 @@
             "hex": "72BE50",
             "source": "https://www.empire.kred"
         },
+        {
+            "title": "Engadget",
+            "hex": "000000",
+            "source": "https://www.engadget.com/",
+            "guidelines": "https://o.aolcdn.com/engadget/brand-kit/eng-logo-guidelines.pdf"
```

```suggestion
            "source": "https://www.engadget.com",
            "guidelines": "https://o.aolcdn.com/engadget/brand-kit/eng-logo-guidelines.pdf"
```

Let's remove that unnecessary slash.

BTW, could you give more details about the guidelines link? Where can we find this link?

---

I removed the trailing slash, although looking at the file, I'd say more than half of the source links have it.
The document was found via the link in this [comment](https://github.com/simple-icons/simple-icons/issues/5309#issuecomment-802714098)